{{cookiecutter.project_name}}/.mlops-setup-scripts/cicd/common/main.tf

```diff
@@ -19,10 +19,18 @@ resource "databricks_secret" "cd_credentials_token" {
   scope        = databricks_secret_scope.cd_credentials.id
 }
 
+resource "databricks_secret" "cd_github_server" {
+  key = "github_server"
+  // regex expression for parsing github server url out of github repo url
+  // example: http(s)://mygitserver.anything/org/repo -> http(s)://mygitserver.anything
+  string_value = regex(".*\/\/[^\/]+", var.github_repo_url)
+  scope = databricks_secrete_scope.cd_credentials.id
+}
+
 resource "databricks_secret" "cd_github_repo" {
   key = "github_repo"
   // Extract the 'organization/repo' substring used to identify the repo
-  string_value = replace(var.github_repo_url, "https://github.com/", "")
+  string_value = replace(var.github_repo_url, databricks_secret.cd_github_server, "")
```

Comment:
I think you'd need to do `databricks_secret.cd_github_server.value` but that also may not work for sensitivity. Could be easier to just pull out the regex logic into a Terraform `local` and pass it into both `resources`.

Reply:
Cool, sounds good.